frontend.py

```diff
@@ -6,39 +6,34 @@
 
 import pyglet
 
-# define the board and size of tiles:
-TILE_WIDTH = 64
-TILE_HEIGHT = 64
-WINDOW_WIDTH = 12*TILE_WIDTH
-WINDOW_HEIGHT = 12*TILE_HEIGHT
 
-
-def init_window(WINDOW_WIDTH, WINDOW_HEIGHT):
+def init_window():
     """
     Return a pyglet window for graphic outputself.
 
     data: a dict created from decoded Tiled 1.2 JSON file
     """
+    from game import WINDOW_WIDTH, WINDOW_HEIGHT
```

Comment:
V Game byla tahle část původně, ale protože tam nemá být žádná logika hry, přesouvalo se to do Frontend

Reply:
Mě to dává větší smysl takhle, protože v game vytváříme proměnnou 'data', z ní lze hned získat čtyři čísla/konstanty, které jsou neměnné po celou hru. Ty pak jenom naimportuju ve frontendu, kde je potřebuju. 
Vytvářet konstanty ve frontendu by znamelo získat proměnnou 'data' pomocí funkce get_data, která potřebuje proměnou 'map_name' a tu získám pouze z modulu game. Ve zkratce zbytečně bych opakovala kód, který je už napsaný v modulu game.